pkg/secretless/entrypoint/start_secretless.go

```diff
@@ -0,0 +1,124 @@
+package entrypoint
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/cyberark/secretless-broker/internal"
+	secretlessLog "github.com/cyberark/secretless-broker/internal/log"
+	"github.com/cyberark/secretless-broker/internal/plugin"
+	"github.com/cyberark/secretless-broker/internal/plugin/v1/event_notifier"
+	"github.com/cyberark/secretless-broker/internal/profile"
+	"github.com/cyberark/secretless-broker/internal/signal"
+	"github.com/cyberark/secretless-broker/pkg/secretless"
+	"github.com/cyberark/secretless-broker/pkg/secretless/config"
+	v2 "github.com/cyberark/secretless-broker/pkg/secretless/config/v2"
+)
+
+// CLIParams holds the command line flag information that Secretless was started
+// with.
+type CLIParams struct {
```

Comment:
Nit: I guess this is my original addition fault but maybe this object should be more generic since it was designed to be used to start secretless from non-CLI usage.